### ISSUE TITLE
When play a DOS game called Framed v1.2, the move and jump is about 4x slow then DOSBox v0.74-3

### ISSUE THREAD

**Issue by wildtang** [joncampbell123/dosbox-x#5205]
### Describe the bug

When play a DOS game called Framed v1.2, the move and jump is about 4x slow then DOSBox v0.74-3

![图片](https://github.com/user-attachments/assets/7607d0fe-eef2-4fed-b0bb-d41d1f45cfab)

### Steps to reproduce the behaviour

1. play a DOS game called Framed v1.2
2. move and jump

### What operating system(s) this bug have occurred on?

Windows 10 LTSC 21H2 (19044.4780)

### What version(s) of DOSBox-X have this bug?

ver.2024.07.01 (VS SDL2 32-bit) Build date/time: Jul 02, 2024 03:45:26am

### Additional information

CPU is AMD Ryzen 7 3700X, Mem is 48G

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
Ran it with default configuration, but I don't think it is significantly slow.
Rather, I don't think it realistic for a casual player to play if it is 4x faster.
Windows VS x64 SDL1 Release 2024.07.01 on Windows 11 22H2.

https://github.com/user-attachments/assets/0908e8b8-5d50-49c1-b7c2-c45446a60ff2

**Comment by wildtang** [comment]
@maron2000 
Yes, I test this game with the default dosbox-x.conf which is in bin\Win32\Release SDL2,
The move and jump speed not slow, is very normal.

Then can someone figure out which config item in these two configuration files cause the speed difference?

**Comment by wildtang** [comment]
I figured it out.

If I changed cycles config from "cycles=auto 300000 max 105%" to the default "cycles             = auto", the move and jump speed became normal. But the game loading time became about 5-6 seconds long when select START NEW GAME.

So the facts are:

1. If use "cycles=auto 300000 max 105%", 

> game loading time is about 1 second when select START NEW GAME, it is fast.
> the move and jump speed are slow.

2. If use "cycles             = auto"
> game loading time is about 5-6 seconds long when select START NEW GAME, it is slow.
> the move and jump speed are normal(not slow).
